IAMOTZ/ievents
# #152797051 Secure privileged API endpoint

#### What does this PR do?
It uses JsonWebToken to add security layer to some API endpoint
#### Description of Task to be completed?
Secure necessary API endpoint
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Create a `.env` file to produce all the environment variables used in `./server/config/config.js`
- run `npm run server:dev`
- the server should start on port 3000
- open postman or any other API testing tool and send a post request to `/api/v1/users` with a body object containing name, email, password, confirm password.
- a data would be return containing a token 
- only with this token can you access `/api/v1/test`
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152797051
#### Screenshots (if appropriate)
None
#### Questions:
None
